scripts/use_gcc82_cvmfs_el7.sh

```diff
@@ -0,0 +1,17 @@
+#---- use gcc and python from SFT in cvmfs
+source /cvmfs/sft.cern.ch/lcg/releases/gcc/8.2.0/x86_64-centos7/setup.sh
+export PATH=/cvmfs/sft.cern.ch/lcg/releases/LCG_96/Python/2.7.16/x86_64-centos7-gcc8-opt/bin:$PATH
+export LD_LIBRARY_PATH=/cvmfs/sft.cern.ch/lcg/releases/LCG_96/Python/2.7.16/x86_64-centos7-gcc8-opt/lib:$LD_LIBRARY_PATH
+
+# --- use correct tbb for ROOT
+export LD_LIBRARY_PATH=/cvmfs/sft.cern.ch/lcg/releases/tbb/2019_U1-5939b/x86_64-centos7-gcc8-opt/lib/:$LD_LIBRARY_PATH 
```

Comment:
Why is this needed ? ROOT compiles TBB internally and is not taking it from CVMFS